visualization/app/codeCharta/ui/copyToClipboardButton/util/clipboardString.ts

```diff
@@ -0,0 +1,21 @@
+import { FileToValue } from "./getFilenamesWithHighestMetrics"
+
+function getStringLineFromItem(item: FileToValue): string {
+	return `\t${String.fromCodePoint(8226)} ${item.name} (${item.value})` + `\n`
+}
+
+function getStringHeaderFromAttribute(title: string): string {
+	return `${title.toUpperCase()}\n`
+}
+
+export function buildTextOfFiles(attributeToFiles: Map<string, FileToValue[]>): string {
+	let clipboardText = ""
```

Comment:
[nitpick]
I think `clipboardText` is too specific as the function doesn't know it will be used in clipboard context later. Maybe just `text` or `result`.

[random info to be ignored]
I personally love a reduce here, which would reduce variable scopes. And in theory a reduce is faster. But as far as I know it is not yet in JS - why I think Ruben doesn't like reduce^^

Reply:
Totally! I love it!

Reply:
Resolved with commit [89ab875](https://github.com/MaibornWolff/codecharta/pull/2942/commits/89ab875127f1d597acf125a17d0178e19e5096b9)